app/main/views/services.py

```diff
@@ -133,6 +133,60 @@ def edit_service(service_id, section_id, question_slug=None):
     )
 
 
+@main.route('/services/<service_id>/edit/<section_id>', methods=['POST'])
+@main.route('/services/<service_id>/edit/<section_id>/<question_slug>', methods=['POST'])
+@role_required('admin-ccs-category')
+def update_service(service_id, section_id, question_slug=None):
```

Comment:
Where do we use this route? I just changed the name of it to `update_service_apocalypse` and nothing broke 🙀

Reply:
The GET route posts to itself.  So the name of the method doesn't matter, but the route is used.

Reply:
There are tests for it, but they call it by URL rather than method name.

Reply:
Gotcha 👍 I guess that's an issue with having the form action buried in ungreppable toolkit templates...